relayer/message_relayer.go

```diff
@@ -247,13 +247,19 @@ func (r *messageRelayer) createSignedMessageAppRequest(requestID uint32) (*avala
 	for node := range nodeValidatorIndexMap {
 		nodeIDs.Add(node)
 	}
+	connectedNodes := r.relayer.network.ConnectPeers(nodeIDs)
 
-	// TODO: We may still be able to proceed with signature aggregation even if we fail to connect to some peers.
-	// 		 We should check if the connected set represents sufficient stake, and continue if so.
-	_, err = r.relayer.network.ConnectPeers(nodeIDs)
-	if err != nil {
+	// Check if we've connected to a stake threshold of nodes
```

Comment:
May want to move this to a `checkSufficientStakeConnected` helper.

Reply:
Modified this to use the existing `utils.CheckStakeWeightExceedsThreshold` helper. I don't think it's worth adding another helper just to calculate the connected stake though.

Reply:
Sounds good